code/modules/materials/Mat_Materials.dm

```diff
@@ -120,6 +120,7 @@
 	proc/addTrigger(var/list/L, var/datum/materialProc/D)
 		for(var/datum/materialProc/P in L)
 			if(P.type == D.type) return 0
+		D.owner = src
```

Comment:
?

Reply:
Fix for Mat_Materials.dm not setting the src.owner, fix was given to me by Pali.

Reply:
🤷